### DISCUSSION TITLE
Custom element always on top of all others

### DISCUSSION THREAD

**Discussion by ed-dantes84** [cdelker/schemdraw#43] [Q&A]
Hello, 

I've been using schemdraw to generate some flowcharts. I've added some custom elements (basically took some references from VISIO). I can't seem to figure out how to correctly place the elements in relation to the others so they follow the standard positioning, they always end up on the top. 

As an example, In the image the element I added is supposed to be directly below "start", followed by 2 paths which look how they are supposed to. The anchors and arrows seem to be working fine. 

![test](https://github.com/cdelker/schemdraw/assets/167686391/534d3ac3-bc10-492f-9ebc-c24571c7338e)
I also attach the way I declared the class:
![test_code](https://github.com/cdelker/schemdraw/assets/167686391/e1aa4327-6717-4e7e-ad72-782fd441be01)

Note: I need the elements to be positioned dynamically, these elements will appear on any position of the flowchart and I don't want them to have an absolute position. 

Thanks for the help.

**Comment by cdelker** [comment]
You may be looking for the `drop` parameter, which defines where to leave the "cursor" after drawing an element. See this example, where `blue.png` is just a 50x50px image filled with blue:

```
class BlueSquare(elm.ElementImage):
    def __init__(self):
        width = height = 1
        super().__init__('blue.png',
                         width=width, height=height,
                         xy=(0, 0))        # xy = Where to place the image in relation to the "cursor"
        self.elmparams['drop'] = (0.5, 0)  # Where to leave the "cursor" after drawing

with schemdraw.Drawing():
    elm.Dot().at((0, 0))
    BlueSquare()
    elm.Dot().color('orange')  # Next element starts at BlueSquare `drop` parameter
```

![image](https://github.com/user-attachments/assets/d148d540-5a53-4daf-ab77-dfdcb9e09020)
